### PR TITLE
doc oc adm must-gather

### PR DESCRIPTION
**Pull request by sanchezl**
Add new `oc adm must-gather` administrative command.

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@bergerhoffer - FYI

**Comment by bergerhoffer** [comment]
Hi @sanchezl, is the must-gather command provided by default with the CLI? I'm testing it now using the CLI build today, and it doesn't look to be available or shown in the help. Has it not officially been added yet? Or is there something that users must do to enable it?

I am planning on adding a section on using must-gather, but am waiting on @deads2k for some initial information.

**Comment by sanchezl** [comment]
Hi @bergerhoffer, the command will be provided by default. It has not been merged as of this moment (https://github.com/openshift/origin/pull/22430).

**Comment by bergerhoffer** [comment]
That would be great @sanchezl! I'll forward you the email that had some questions I had on it.

And I'll watch that PR and will merge this PR once that gets in. Thanks for adding this!

File: modules/cli-administrator-other.adoc [hunk 1]
```diff
@@ -38,6 +38,15 @@ $ oc adm config view
 $ oc adm config use-context test-context
 ----
 
+== must-gather
+
+Gather debugging information.
+
+.Example: Gather debugging information
```
Comment (sferich888) [inline modules/cli-administrator-other.adoc]: Consider reviewing https://docs.google.com/document/d/1AN0yobTx9YsnwfGdoGjLywx2gfvEL3U6E8Oy8RgjJFA/edit?ts=5cb47890 for some context as to why you want this.

**Review by bergerhoffer — COMMENTED** [review]
@sanchezl So now that I see the PR was merged and can test, I see that must-gather is listed in the help under cluster management (not other commands). Can you move this to the cli-administrator-cluster-management.adoc file [1] instead (alphabetically)? Thanks!

[1] https://github.com/openshift/openshift-docs/blob/enterprise-4.0/modules/cli-administrator-cluster-management.adoc

**Comment by bergerhoffer** [comment]
Closing this PR in favor of #14811.

**Pull request closed without merging**